external/TopQuarkAnalysis/TopQuarkAnalysis

```diff
@@ -0,0 +1 @@
+BristolAnalysis/Tools/external/TopQuarkAnalysis
```

Comment:
at somepoint I did git add --all. this must have been included. It is not
of any importance

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

 Douglas.Burns@bristol.ac.uk <John.Newby@bristol.ac.uk>

On 20 November 2015 at 15:06, EmyrClement <notifications@github.com> wrote:

> In external/TopQuarkAnalysis/TopQuarkAnalysis
> <https://github.com/BristolTopGroup/AnalysisSoftware/pull/189#discussion_r45471273>
> :
>
> > @@ -0,0 +1 @@
> > +BristolAnalysis/Tools/external/TopQuarkAnalysis
>
> Not sure what this change is?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BristolTopGroup/AnalysisSoftware/pull/189/files#r45471273>
> .
>